src/Paket/Paket.fsproj

```diff
@@ -30,6 +30,8 @@
     <StartArguments>update</StartArguments>
     <StartAction>Project</StartAction>
     <StartProgram>paket.exe</StartProgram>
+    <StartWorkingDirectory>C:\projects\SourceLink</StartWorkingDirectory>
```

Comment:
i really hope so, see Microsoft/visualfsharp#496 , the PTVS has it. 
We can import PTVS code or use a new common project system
Pls @forki add yours missing features and bugs to issue, i'll check if PTVS supports it already